packages/nx/src/daemon/server/server.ts

```diff
@@ -153,7 +158,9 @@ const server = createServer(async (socket) => {
   });
 
   socket.on('close', () => {
```

Comment:
I thought this only gets called when the client is destroyed?

The client won't be destroyed in the plugin worker right? Does this actually do anything at the moment?